announcements/_posts/2019-02-28-mentors-meeting.md

```diff
@@ -0,0 +1,9 @@
+---
+title:  Informative Meeting for Mentors, 1 March 2019
```

Comment:
Just a small point - say "GSoC Mentors" as this will appear as its own banner and people need the context.

Reply:
Done